pkg/buildkitutil/buildkitutil_unix.go

```diff
@@ -0,0 +1,50 @@
+//go:build freebsd || linux
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package buildkitutil
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/containerd/log"
+	"github.com/containerd/nerdctl/pkg/rootlessutil"
+)
+
+func getBuildkitHostCandidates(namespace string) ([]string, error) {
+	if namespace == "" {
+		return []string{}, fmt.Errorf("namespace must be specified")
+	}
+	// Try candidate locations of the current containerd namespace.
+	run := "/run/"
```

Comment:
Ain't it `/var/run` on FreeBSD?

Reply:
I have no idea, I just moved this code block over from the other file. If you'd asked me yesterday, I'd have said it was `/var/run` on Linux too, having grown up in a pre-FHS3.0 world.

Reply:
`/var/run` seems safer on non-Linux